src/FSharp.Stats/DistanceMetrics.fs

```diff
@@ -88,6 +88,77 @@ module DistanceMetrics =
                 if not (isNan x) then
                     dist <- dist + System.Math.Abs x
             dist
+
+        /// <summary>The [Minkowski distance](https://en.wikipedia.org/wiki/Minkowski_distance) between two vectors (ignores nan), of order `p`.</summary>
+        /// <remarks>The two vectors need not have equal lengths: when one vectors is exhausted any remaining elements in the other vectors are ignored.</remarks>
+        /// <param name="s1">first vector</param>
+        /// <param name="s2">second vector</param>
+        /// <param name="p">integer constrained to `p >= 1`</param>
+        /// <returns>Minkowski distance between elements of given vectors</returns>
+        /// <example> 
+        /// <code> 
+        /// // e.g. v1 and v2 initialization
+        /// let v1 = vector [3.14; 2.0; 3.1]
+        /// let v2 = vector [9.1; 2.5; 3.7]
+        /// 
+        /// // Apply the minkowski distance to v1 and v2
+        /// Vector.minkowski v1 v2 3
+        /// </code> 
+        /// </example>
+        let inline minkowski (v1: Vector<'a>) (v2: Vector<'a>) (p: int) : float option =
+            if p < 1 then
+                None
+            else
+                let dim = min v1.Length v2.Length
+                let mutable acc = 0.0
+
+                for i in 0 .. (dim - 1) do
+                    let abs = 
```

Comment:
Is there a reason you didnt use `abs (v1.[i] - v2.[i])` to determine the absolute difference?